build-images.sh

```diff
@@ -45,13 +45,6 @@ main() {
     gh release create --title "$tag (build $(date +"%Y-%m-%d"))" "$tag"
   fi
   gh release upload --clobber "$tag" "${assets[@]}"
-
-  gh release view --json assets | jq -r ".assets | map(.name) | .[] | select(test(\"$arch\"))" >"$TMP/existing-assets"
```

Comment:
Looks like this filter didn't quite work. We can also delete stuff manually otherwise.